crates/net/downloaders/src/headers/linear.rs

```diff
@@ -787,19 +793,27 @@ impl LinearDownloadBuilder {
         self
     }
 
+    /// Set the latest canonical header we have downloaded.
+    pub fn set_local_head(mut self, local_head: SealedHeader) -> Self {
```

Comment:
stick to the naming convention of existing functions and drop `set_`

Reply:
Removed since `update_local_head`/`update_sync_target` exist on `main`